### ISSUE TITLE
phenotype_value.assoc.txt

### ISSUE THREAD

**Issue by fanmiaowang** [voichek/kmersGWAS#127]
Hi Voichek,

GWAS outputs phenotype_value.assoc.txt (GEMMA results for all k-mers passing the first filtering of the pipeline). What is exactly the first filtering? Please explain a bit more if it is possible. 
And do you have any idea why I did not get output of "phenotype_value.assoc.txt.gz" - all the GEMMA results? Therefore, the Pvalue for some kmers are lacking, although they are kmers "not passing the first filtering". Thanks.

**Comment by voichek** [comment]
Hi,

1. The filtering scheme is explained in great detail in the method section of the paper with an accompanied supplementary figure. 
2. If `kmers/output/phenotype_value.assoc.txt.gz` is empty something clearly went wrong. Please first run example 1 to make sure there isn't any issue with your system or how you made the k-mers-table.

Best,
Yoav

**Comment by fanmiaowang** [comment]
Hi Yoav,

Thank you for your explaination. 

1. I will find the answer from the method section of your paper.

2. Well, I tried to run example 1. I did get the output in kmers/output/phenotype_value.assoc.txt.gz. However the phenotype_value.assoc.txt.gz only contains 10001 kmers with pvalues provided. There is no output for all kmers (before the first filtering). If this is the case, I am relieved. But if not, I need to figure out what is going wrong.

Best,
Fanmiao
